docs/web_advanced.rst

```diff
@@ -403,6 +404,16 @@ This is required for type safety (e.g. when checking with mypy)::
         data = request.app[my_private_key]
         # reveal_type(data) -> str
 
+.. note::
+
+   If warnings are explicitly enabled (e.g. ``-We`` or ``-Wall``) then
+   using :class:`str` keys will trigger a warning. Like other warnings
+   this can be filtered out easily if you have no interest in this feature.
```

Comment:
pet peeve! 'easy' should never be used in documentation; things are easy when you know how to do them, if you are reading documentation then this is prior to you knowing how to do them :)

Reply:
That's a good point! I occasionally hear tech writers mentioning this, but it's so often overlooked. We need to do better!

Reply:
I don't really mind either way, but I personally don't get it. It's easy in the context of the documentation, as you only need to copy a line of code or CLI argument. Plenty of things are still difficult after you know how to do them. The word doesn't add anything of value to the docs though, so it makes sense to remove it for that reason, if we do merge it in.

Reply:
@Dreamsorcerer it appears we failed to convey the complete context of the sentiment.
The point here is not that it's _objectively_ easy or hard in the context, but how it affects the receiving side of the text (the reader).
When people hear (see) certain words, they have an effect on their perception. It's a subconscious thing. Hearing “just” often makes people _feel_ dumb. They get defensive and don't accept the information as easily. This is not the recommendation for docs only, it's applied to all the teaching contexts (tutoring/mentoring/training/coaching/public speaking etc.)

The solution that is typically suggested is avoiding the use of such trigger words (or phrases). The same goes for saying “easy” / “straightforward” / “obvious” / “intuitive”. All of these have a great potential to discourage and hurt self-confidence of the readers. The less experience one has, the more harmful it is. This is a straight way to diminish people's feeling of achievement.

Instead, of using these, it's recommended to just skip to the explanation, without showing how much the writer is smarter than a reader.